pkgs/by-name/cl/clock-rs/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "clock-rs";
+  version = "0.1.213";
+
+  src = fetchFromGitHub {
+    owner = "Oughie";
+    repo = "clock-rs";
+    rev = "v${version}";
```

Recently a new arg `tag` was introduced, that should be used over rev, when you specifically want to fetch a tag. This avoids confusions with version branches.

```suggestion
    tag = "v${version}";
```